src/types.ts

```diff
@@ -175,43 +175,45 @@ export type Config = {
 
 export type Asset = { [key: string]: any };
 
-export type Assets = {
-  actions: Asset[];
-  attackProtection: Asset;
-  branding: Asset;
-  clients: Asset[];
-  clientGrants: Asset[];
-  connections: Asset[];
-  databases: Asset[];
-  emailProvider: Asset;
-  emailTemplates: Asset[];
-  guardianFactorProviders: Asset[];
-  guardianFactors: Asset[];
-  guardianFactorTemplates: Asset[];
+export type Assets = Partial<{
```

Comment:
Because `context.assets` can have undefined properties due to omission in the config files, the types should reflect that possibility with the `Partial` utility type.